.github/actions/check-license/action.yml

```diff
@@ -0,0 +1,19 @@
+name: Check License
+description: Checks that the committed third party licenses file is up to date
+
+inputs:
+  licensefile:
+    description: 'A relative path from the repository root to the 3rd party license file'
+    default: 'THIRD-PARTY-LICENSES.txt'
+    required: false
```

Comment:
> Should this be required (what would this do without a licensefile as input?)

The default value, line 7, is used if the input parameter is not explicitly specified by the caller.

I added this just in case some repo has a different name and/or location for the third party license file, allowing the caller to override if so.

Might be overkill, as I think all our repos have the same filename in the same location.